README.md

```diff
@@ -240,6 +240,7 @@ Name | Website | Region
 [FlightAware](/company-profiles/flightaware.md) | https://flightaware.com | USA
 [Flip](/company-profiles/flip.md) | https://flip.id | Indonesia
 [Flowing](/company-profiles/flowing.md) | https://flowing.it | Italy
+[Fly.io](/company-profiles/fly-io.md) | https://fly.io | USA
```

Comment:
Due to https://fly.io/jobs/ and `Current openings are for remote positions anywhere in the world.`, this probably should be `Worldwide`? or is the anywhere tag 'Anywhere within the USA'?

Reply:
Yeah, I think you are right, I will change this, thank you =D